client/src/pages/Explorer.js

```diff
@@ -73,47 +98,82 @@ function FavoriteCheckBox() {
 
 const ExplorerPage = () => {
   const classes = useStyles();
+  const [generalData, setGeneralData] = useState({places: [], favorites: []});
+  const citiesUrl = "/api/cities/us/?population=1500000";
+  const userFavoriteCitiesUrl = "/api/users/favoriteCities";
+  useEffect(() => {
+    async function getData(){
+      try {
+        const result1 = await fetch(userFavoriteCitiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          },
+        })
+        const result2 = await fetch(citiesUrl, {
+          method: "GET",
+          headers: {
+            "Content-Type": "application/json",
+          }
+        })
+        const places = await result2.json();
+        const favorites = await result1.json();
+        userFavoritePlaces = Array.isArray(favorites) ? favorites : [];
+        setGeneralData({
+          places: Array.isArray(places) ? places : [],
+          favorites: Array.isArray(favorites) ? favorites : []
+        });
+      } catch (error) {
+         console.log();
+      }
+    }
+    getData();
+  }, [userFavoriteCitiesUrl, citiesUrl]);
   return (
-		<Container className={classes.pageContainer}>
-			<Typography variant="h4" className={classes.title}>
-				Explore destinations
-			</Typography>
-			<Typography
-				variant="subtitle2"
-				className={classes.title}
-				style={{ color: "#c5bec4", fontSize: 12 }}
-			>
-				World's top destinations to explore
-			</Typography>
-			<Grid container spacing={3} justify="center" style={{ marginTop: 32, height: "75%" }}>
-				{places.map((place) => (
-					<Grid item key={place.countryName} xs={12} sm={3}>
-						<div
-							className={classes.paperContainer}
-							style={{
-								backgroundImage: `linear-gradient(to bottom, transparent, rgba(52, 52, 52, 0.63)), url(${place.imageUrl})`,
-								// height: "100%",
-							}}
-						>
-							<div className={classes.bottomInformationContainer}>
-								<span className={classes.bottomInformationSubContainer1}>
-									<span style={{ fontSize: 17, color: "white" }}>
-										{place.townName},
-									</span>
-									<span style={{ fontSize: 11, color: "rgb(175 175 175)" }}>
-										{place.countryName}
-									</span>
-								</span>
-								<span className={classes.bottomInformationSubContainer2}>
-									<FavoriteCheckBox />
-								</span>
-							</div>
-						</div>
-					</Grid>
-				))}
-			</Grid>
-		</Container>
-	);
+    <Container className={classes.pageContainer}>
+      <Typography variant="h4" className={classes.title}>
+        Explore destinations
+      </Typography>
+      <Typography
+        variant="subtitle2"
+        className={classes.title}
+        style={{ color: "#c5bec4", fontSize: 12 }}
```

Comment:
Let's not mix inline styling and className. We're gonna stick to `className` for consistency